### PR TITLE
Fix livesync when "livesync android" is executed and iOS simulator is runinng

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

File: services/usb-livesync-service-base.ts [hunk 1]
```diff
@@ -48,7 +48,8 @@ export class UsbLiveSyncServiceBase implements IUsbLiveSyncServiceBase {
 		beforeBatchLiveSyncAction?: (_filePath: string) => IFuture<string>,
 		iOSSimulatorRelativeToProjectBasePathAction?: (projectFile: string) => string): IFuture<void> {
 		return (() => {
-			let synciOSSimulator = this.$hostInfo.isDarwin ? this.$iOSEmulatorServices.isSimulatorRunning().wait() || (this.$options.emulator && platform.toLowerCase() === "ios") : false;
+			let platformLowerCase = platform.toLowerCase();
+			let synciOSSimulator = this.$hostInfo.isDarwin ? platformLowerCase !== "android" && (this.$iOSEmulatorServices.isSimulatorRunning().wait() || (this.$options.emulator && platformLowerCase === "ios")) : false;
```
Comment (rosen-vladimirov) [inline services/usb-livesync-service-base.ts]: Why not
```TypeScript
let synciOSSimulator = this.$hostInfo.isDarwin() &&  platformLowerCase === "ios" && (this.$iOSEmulatorServices.isSimulatorRunning().wait() || this.$options.emulator);
```

Comment (rosen-vladimirov) [inline services/usb-livesync-service-base.ts]: also make sure to add  `()` after `hostInfo.isDarwin` - this is method ;)

Comment (Fatme) [inline services/usb-livesync-service-base.ts]: `hostInfo.isDarwin` - is a property - https://github.com/telerik/mobile-cli-lib/blob/master/declarations.d.ts#L338 not a function

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**